oap-server/server-starter/src/main/resources/telegraf-rules/default.yaml

```diff
@@ -0,0 +1,50 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+expSuffix: service(['host'], Layer.OS_LINUX)
```

Comment:
Is this file working well as same as OTEL Linux monitoring? Could we activate them at the same time?

Reply:
> Is this file working well as same as OTEL Linux monitoring? Could we activate them at the same time?

I am not sure whether they can activate at the same time, I will test it.

Reply:
If it can't for now, please let me know. Unless there are differences at aggregation function level, or downsampling. We should be able to.

Otherwise, this configurations(rules) should be in default release. Because there is no dashboard for it.